Tests/APITesters/RevenueCatUIAPITester/SwiftAPITester/PaywallViewControllerAPI.swift

```diff
@@ -43,6 +45,7 @@ func paywallFooterViewControllerAPI(_ delegate: Delegate, _ offering: Offering?)
 
     let _: UIViewController = PaywallFooterViewController(offering: offering)
     let _: UIViewController = PaywallFooterViewController(offeringIdentifier: "offering")
+    let _: UIViewController = PaywallFooterViewController(offeringIdentifier: "offering", fontName:"Papyrus")
```

Comment:
Sorry another lint issue:
```suggestion
    let _: UIViewController = PaywallFooterViewController(offeringIdentifier: "offering", fontName: "Papyrus")
```